### DISCUSSION TITLE
actualize_script.php mistakenly errors that the cron call as not invoked from command line

### DISCUSSION THREAD

**Discussion by trickjarrett** [FreshRSS/FreshRSS#6883] [Q&A] [answered]
### Describe the bug

I have a cron job scheduled to run:

`
` php /home/XXXX/public_html/reader/app/actualize_script.php
`

My host does not support me using 'www-data' which is what I assume is the core of this issue.

When the cron job runs, I get the error:

> FreshRSS error: This PHP script may only be invoked from command line!

### To Reproduce

1. Go to '…'
2. Click on '…'
3. Scroll down to '…'

### FreshRSS version

1.23.1

### Environment information

- Database version: 10.6.19-MariaDB-log
- PHP version: PHP 8.1
- Installation type: Cpanel / source
-Web server type: Apache
- Device: N/A
- OS: Linux
- Browser: N/A

**Comment by trickjarrett** [comment, marked as answer]
I found a work around. The use of the direct php instantiation path:

` /usr/local/bin/php /home/XXXX/public_html/reader/app/actualize_script.php

**Comment by Alkarex** [comment]
It could be that your PHP instance calling `actualize_script.php` is not running in CLI mode (command line)
